python/fastquant/strategies/base.py

```diff
@@ -289,6 +294,33 @@ def next(self):
                         self.sell(exectype=bt.Order.StopTrail, trailpercent=self.stop_trail, size=final_size)
 
         elif self.sell_signal():
+            if self.allow_short == True:
+
+                # Sell short based on the closing price of the previous day
+                if self.execution_type == "close":
+
+                    if self.transaction_logging:
+                        self.log("SELL CREATE, %.2f" % self.dataclose[0])
+
+                    sell_prop_size = int(self.short_max * 
```

Comment:
```suggestion
                    sell_prop_size = int(SELL_PROP * 
```

Reply:
Let's use sell prop as basis here. I understand short max is to ensure that the short sell is not too large above a specific number. I think this is better set as a constant, so something like `self.short_max * self.init_cash`